test/src/evaluation.spec.ts

```diff
@@ -408,9 +408,10 @@ describe('Evaluation specs', function () {
           return (error = error_);
         });
       expect(error).toBeTruthy();
-      expect(error.message).toContain(
-        'JSHandles can be evaluated only in the context they were created'
-      );
+      expect(error.message).atLeastOneToContain([
+        'JSHandles can be evaluated only in the context they were created',
+        "Trying to evaluate JSHandle from different frames. Usually this means you're using a handle from a page on a diffrent page.",
```

Comment:
```suggestion
        "Trying to evaluate JSHandle from different frames. Usually this means you're using a handle from a page on a different page.",
```

Can we keep the same error message?

Reply:
We can, but this new error message is more explicit. You mind if we change the CDP one instead?

Reply:
Let's keep both for now, and do it later.